app/api/main.py

```diff
@@ -1,9 +1,18 @@
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 from app.api.routers.anonymize_file import anonymize_router
 from app.api.routers.ask_gpt import ask_router
 
 app = FastAPI()
 
+app.add_middleware(
```

Comment:
this code is already on `main` branch. you should do another rebase

Reply:
okay but idk why even tho I did I rebase !